### PR TITLE
Create more rolebindings and CSVs during setup

### PR DESCRIPTION
**Pull request by rajivnathan**
- Adds a prepare step that installs one of the available operators that works at the "All Namespaces" scope. I chose Kiali since it seems pretty lightweight and I couldn't get the OSD route-monitor operator working but we could use any operator that fills this need. Maybe once we onboard SBO we can use that since with the improvements it has a small footprint and will be part of the Sandbox experience which is better than using another operator which can be an external influence on the test results
- Adds 12 RoleBindings to bring the resource numbers w/3000 users in line with production
- Removes an unnecessary secret

### CODE REVIEW

File: setup/operators/install_operators.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	AllNamespacesOperatorName   = "kiali-ossm"
+	AllNamespacesOperatorPrefix = "kiali-operator"
+)
+
+func EnsureAllNamespacesOperator(cl client.Client, csvNamespace string) error {
```
Comment (alexeykazakov) [inline setup/operators/install_operators.go:19]: Let's add a comment explaining why we install operators here.

**Review by alexeykazakov — APPROVED** [review]
I wonder if really care if the router operator works or not. I guess just installing it would do the trick of creating the needed amount of CSVs?

But anyway I'm fine with it sing kiali for now. But let's add some comments in the code to explain why we are doing it.

**Review by MatousJobanek — APPROVED** [review]
Cool :+1: 
BTW, how about adding a note to the README to make it transparent? https://github.com/codeready-toolchain/toolchain-e2e/tree/master/setup#dev-sandbox-setup-1 it's just a proposal - I don't insist on it

File: setup/operators/install_operators.go (hunk 1 above)
Comment (rajivnathan) [reply setup/operators/install_operators.go:19]: Sure, I had a comment in root.go but it's worth having one here too. Done 1bbdd9c38f46e6cad22e261315e7f2b837b72b90

**Comment by rajivnathan** [comment]
@alexeykazakov Sorry, should have been more clear. I meant I couldn't get the route operator to create the CSVs so that's why I went with kiali.

**Comment by rajivnathan** [comment]
@MatousJobanek Good point about making it more transparent. Added a note about it: ef8f694806203ddf2f4ab934911a4da3d0ac1a14

**Pull request merged**